ghost/signup-form/vite.config.ts

```diff
@@ -42,7 +49,9 @@ export default (function viteConfig() {
                 output: {}
             },
             commonjsOptions: {
-                include: [/packages/, /node_modules/]
+                include: [/ghost/, /packages/, /node_modules/],
```

Comment:
```suggestion
                include: [/ghost/, /node_modules/],
```

I don't think we need `packages` any longer? 🤔 We only use `/ghost/` in this repo